core/string/translation.cpp

```diff
@@ -1130,7 +1130,7 @@ StringName TranslationServer::_get_message_from_translations(const StringName &p
 	bool near_match = false;
 
 	for (const Set<Ref<Translation>>::Element *E = translations.front(); E; E = E->next()) {
-		const Ref<Translation> &t = E->get();
+		Ref<Translation> &t = const_cast<Ref<Translation> &>(E->get());
```

Comment:
Is it possible to remove `const` from `const Set<Ref<Translation>>::Element *E` here? Then you probably wouldn't need to `const_cast`.

Reply:
No, `call` is non-const function so I cannot call it otherwise.